r/tests/testthat/test-dplyr.R

```diff
@@ -156,6 +156,21 @@ test_that("select/rename", {
   )
 })
 
+test_that("select/rename with selection helpers", {
+
+  # TODO: add some passing tests here
+
+  expect_error(
+    expect_dplyr_equal(
+      input %>%
+        select(where(is.numeric)) %>%
```

Comment:
is "where" the problem or is it "is.numeric"?

Reply:
It's both. In our `select()` and `rename()` methods, we're using some "questioning-stage" rlang functions (namely `vars_select` and `vars_rename` and in those it fails on `where`. In the `relocate` method I added, it fails to find the column types.